warp-finn/src/main/res/values/finn_colors.xml

```diff
@@ -65,10 +65,10 @@
 
     <color name="gray50">#F6F6F6</color>
     <color name="gray100">#F0F0F2</color>
-    <color name="gray200">#D1D1D6</color>
-    <color name="gray300">#BBBBC2</color>
-    <color name="gray400">#A5A5AD</color>
-    <color name="gray500">#80808A</color>
+    <color name="gray200">#DEDEE3</color>
```

Comment:
New tori and finn colors?

Reply:
They have been updated a while back, i noticed that i forgot to change the values in xml. Compose was fine :)